### PR TITLE
Configuration files for GrafanaCon 2022 presentation.

### PR DESCRIPTION
**Pull request by pstibrany**
Configuration files used for GrafanaCon 2022 presentation deploying Mimir via Helm charts.

### CODE REVIEW

**Comment by pstibrany** [comment]
@jdbaldry @osg-grafana If we put these files into `docs/sources/tutorials/grafanacon-2022/` are they going to end up on Grafana public website?

**Comment by jdbaldry** [comment]
> @jdbaldry @osg-grafana If we put these files into `docs/sources/tutorials/grafanacon-2022/` are they going to end up on Grafana public website?

Yes, content in the `docs/sources` directory is published to the website as part of the https://github.com/grafana/mimir/blob/main/.github/workflows/publish-technical-documentation-next.yml workflow.

Content in the `tutorials` subdirectory is treated specially and is moved to the `/tutorials` section of the website (https://grafana.com/tutorials/).

Is this what you want?

**Comment by pstibrany** [comment]
> > @jdbaldry @osg-grafana If we put these files into `docs/sources/tutorials/grafanacon-2022/` are they going to end up on Grafana public website?
> 
> Yes, content in the `docs/sources` directory is published to the website as part of the https://github.com/grafana/mimir/blob/main/.github/workflows/publish-technical-documentation-next.yml workflow.
> 
> Content in the `tutorials` subdirectory is treated specially and is moved to the `/tutorials` section of the website (https://grafana.com/tutorials/).
> 
> Is this what you want?

Not quite. I want to include these config files *somewhere* in Mimir repository so that people watching GrafanaCon presentation can find them, but I don't think they should be on https://grafana.com/tutorials website.

What would be a better place to store these config files?

Some ideas:

- docs/configurations/grafanacon-2022
- docs/internal/... ?

(I'd like to mention the location of these config files in the talk itself.)

**Comment by jdbaldry** [comment]
I think `docs/configurations` would make a lot of sense, that's where we keep a lot of configuration files that are related to docs or demonstrations.

File: docs/configurations/grafanacon-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Before you start
+
+**Warning:** Following commands will not specify explicit context for `kubectl` commands. Make sure to select correct
+context and namespace. For example for Docker for Desktop comes with `docker-desktop` context preinstalled. We will
```
Comment (pracucci) [inline docs/configurations/grafanacon-2022/index.md:4]:
```suggestion
context and namespace. For example, Docker for Desktop comes with `docker-desktop` context preinstalled. We will
```

File: docs/configurations/grafanacon-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+use `default` namespace in this demo.
+
+```
+kubectl config use-context docker-desktop
+kubectl config set-context docker-desktop --namespace=default
+```
+
+# Preparation
+
+helm repo add grafana https://grafana.github.io/helm-charts
```
Comment (pracucci) [inline docs/configurations/grafanacon-2022/index.md:14]: Please wrap commands with a code block.

_Feedback applies below too_

File: docs/configurations/grafanacon-2022/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+# Install Ingress
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.2/deploy/static/provider/cloud/deploy.yaml
+
+# Grafana Agent Operator
+
+Links:
+
+- https://grafana.com/docs/agent/latest/operator/helm-getting-started/
+- https://grafana.com/docs/agent/latest/operator/custom-resource-quickstart/
```
Comment (pracucci) [inline docs/configurations/grafanacon-2022/index.md:26]: Link them too (markdown link)

**Review by pracucci — APPROVED** [review]

**Pull request merged**